web/app/themes/imb/header.php

```diff
@@ -1,36 +1,42 @@
 <?php get_template_part( 'head' ); ?>
-<header class="banner" role="banner">
-	<div class="container navbar-expand-md navbar navbar-default navbar-static-top flex-wrap">
-		<div id="branding" class="flex-fill">
-			<a href="<?php echo home_url(); ?>">
-				<img alt="IMB Logo" src="<?php bloginfo( 'template_directory' ); ?>/assets/img/IMBLogo_2X.png" />
-			</a>
-		</div>
 
-		<div id="search" class="align-self-start flex-fill flex-shrink-0">
-			<?php get_search_form(); ?>
-		</div>
+<body <?php body_class(); ?>>
 
-		<div class="navbar-header">
-			<button class="navbar-toggler" type="button" data-toggle="collapse" data-target=".navbar-collapse"
-					aria-controls=".navbar-collapse" aria-expanded="false" aria-label="Toggle navigation">
-				<span class="icon-bar"></span>
-				<span class="icon-bar"></span>
-				<span class="icon-bar"></span>
-			</button>
-		</div>
+<?php do_action('after_body_open_tag'); ?>
+
+<div class="ccfw-banner-background">
```

Comment:
@BNewing the ccfw banner div seems to now wrap the whole page. Just checking if this is correct as it seemed abit odd to have a banner div.

Reply:
The `ccfw-banner-background` div should wrap the rest of the page content, but the cookie banner itself shouldn't. I was aiming for this:

https://www.josephearl.co.uk/post/css-overlay/

It isn't properly working at the moment - I had it working at one point, but must have changed something in between page reloads which is annoying! I made a reference to that on the PR for the banner on the cookie repo but should have put it here - got mixed up with what went where.